utils/data_interaction.py

```diff
@@ -0,0 +1,12 @@
+from io import StringIO
+import requests
+import pandas as pd
+
+def takeKeywords():
```

Comment:
This is gives a list of words for the ML to crunch? From the app user's perspective its the sentence entered broken by space?

Reply:
Yes, it'll take a list of keywords. Users are expected to enter a bunch of words without grammar like "when, birthday" or "what, eat, yesterday". I can add a method to check whether user inputs correctly

Reply:
Okay thanks. this is good to know. So on the app side I turn their input into an array before returning the output

Reply:
Exactly.